requirements.txt

```diff
@@ -17,3 +17,5 @@ sklearn
 sympy
 tensorflow
 xgboost
+torch
+torchvision
```

Comment:
Please keep the filenames in `requirements.txt`sorted to reduce the likelihood of duplicate entries.